src/services/posts/fetch-posts.js

```diff
@@ -1,7 +1,6 @@
 const db = require("../../db")
 
-const fetchPosts = async(isFlagged) => {
-
+const fetchPosts = async({isFlagged, cursor, limit }) => {
```

Comment:
We can handle the default at the service level since that's where the main fetch is happening. That could help to prevent having to pass limit of 20 to the function calls in the tests

```suggestion
const DEFAULT_LIMIT = 20

const fetchPosts = async({isFlagged, cursor, limit = DEFAULT_LIMIT}) => {
```

Reply:
yes please